### PR TITLE
More Austrian Changes

### PR DESCRIPTION
**Pull request by ccfox**
I forgot West Austria was a tag, so I fixed up those parties.

I also made election events for Austria, though not all the events I want to make are finished yet. Primarily wanted to make them because I did something different, and I might go back and change it for other nations I've done. Rather than make each election its own event, I only make significant changes in politics or significant elections their own events - with the rest being generic and having a selection of the prominent parties, with 'off-ramps' to alt-history if you continuously elect a party. It seems like it would save on events.

And finally I also made a ton of loc fixes with the characters - and '' not showing up correctly in the loc.

### CODE REVIEW

File: CWE/events/Austria - Historic.txt [hunk 1]
```diff
@@ -0,0 +1,698 @@
+$1949 Election
```
Comment (Outta-my-way-burrocrat) [inline CWE/events/Austria - Historic.txt:1]: typo

Comment (ccfox) [reply CWE/events/Austria - Historic.txt]: Fixed, thanks.

**Comment by ccfox** [comment]
Since we now have a release date for Victoria 3, I'm going to apply the election consolidation changes to the GDR, and then mark this as ready for review since I don't know if I'll finish with the rest of the events I want before then.

File: CWE/localisation/Chrisfox's events.csv [hunk 2]
```diff
@@ -54,63 +54,63 @@ EVTDESC8012111;The DPRK has an interesting ideology - it incorporates elements o
...
+EVTDESC8023012107;With power in Romania now passing to anti-Stalinists, the time has come to purge the late Soviet leader's influence from within the party and Romanian society. Ana Pauker - initially seen as supportive of the direction of the new government before turning out of its favor with increasingly vocal opposition - has today announced a campaign of liberalization within the party, with even more far-reaching plans for strengthening the 'Moscow Faction' influence.;;;;;;;;;;;;;x;;;;;;;;;;;;;;;;
+EVTDESC8023012108;The 'Magyar Wing' of Romania's Communist Party was dissolved in 1953, just a year after the creation of the 'Magyar Autonomous Region' in Romania. It had provided one of the few mechanisms for Hungarians in Romania to fight against the increasing policy of 'Romanianization' from the Nationalists and Stalinists that had helped prop up the rule of Gheorghiu-Dej. \n\nGiven that the Stalinists no longer wield much political influence - the question has been brought up by more liberal-minded party members as to a potential revival of this institution.;;;;;;;;;;;;;x;;;;;;;;;;;;;;;;
```
Comment (settintotrieste) [inline CWE/localisation/Chrisfox's events.csv:64]: I think the localisation file has been encoded wrongly. It should be set as ANSI.

File: CWE/events/Austria - Historic.txt [hunk 3]
```diff
@@ -0,0 +1,698 @@
...
+		}
+	}
+
+	option = {
+		name = "Social Democrats manage to gain majority over their coalition."
+
+		ruling_party_ideology = socialist
+
+		any_pop = { 
+			ideology = { factor = 0.2 value = socialist }
```
Comment (settintotrieste) [inline CWE/events/Austria - Historic.txt:58]: Effects adding socialist ideology should exclude rich strata.

File: CWE/localisation/Chrisfox's events.csv (hunk 2 above)
Comment (ccfox) [reply CWE/localisation/Chrisfox's events.csv:64]: That's strange, opening the file in Notepad++ says that its already in ANSI, even when I download a 'fresh' version of the file.

File: CWE/events/Austria - Historic.txt (hunk 3 above)
Comment (ccfox) [reply CWE/events/Austria - Historic.txt:59]: Alright, that should all be fixed now.

**Pull request merged**